serverless/src/shared/buildHierarchicalCsvPaths.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @file buildHierarchicalCsvPaths.js
+ * @description Builds hierarchical CSV paths for a given scheme and concept map.
+ */
+
+import { cloneDeep } from 'lodash'
+
+import { formatCsvPath } from '@/shared/formatCsvPath'
+import { getNarrowers } from '@/shared/getNarrowers'
+import { isCsvLongNameFlag } from '@/shared/isCsvLongNameFlag'
+import { isCsvProviderUrlFlag } from '@/shared/isCsvProviderUrlFlag'
+
+/**
+ * Builds hierarchical CSV paths recursively.
+ *
+ * @param {number} csvHeadersCount - The number of CSV headers.
```

Comment:
can you use continue to generate examples of what these values look like?

Reply:
Done